### DISCUSSION TITLE
"Move" Doesn't Snap to Dock Correctly

### DISCUSSION THREAD

**Discussion by aron-gergely** [rxhanson/RectanglePro-Community#224] [Q&A] [answered]
Hi All!
I'm in LOVE with this app, but my snapping is bugged when I use the "Move" function within the "Window Throw" menu. It's only happening when the dock is at the bottom. Do you guys know any solution to this? 

Here is a video to demonstrate the problem: https://youtu.be/S1zj45yGixo

Thank you in advance!

**Comment by rxhanson** [comment, marked as answer]
Thanks for reporting & the video. I put in the fix and it will go in the next release.

**Reply by aron-gergely** [reply]
Thank you so much for the fix and the quick reply! :))

**Reply by rxhanson** [reply]
This is now in the latest release, let me know if you hit any issues with it.
